### PR TITLE
Anek Telugu: Version 1.003 added

### PR DESCRIPTION
**Pull request by yanone**
740dc41: [gftools-packager] Anek Telugu: Version 1.003 added

* Anek Telugu Version 1.003 taken from the upstream repo https://github.com/EkType/Anek at commit https://github.com/EkType/Anek/commit/634384abf35544275862ae22c7da44cc9c916caa.

 f43908b: [gftools-packager] ofl/anektelugu remove METADATA "source".  google/fonts#2587

### CODE REVIEW

**Comment by RosaWagner** [comment]
@yanone can you check if there is not a missing glyphs?
possible related issue: https://github.com/googlefonts/fontbakery/issues/3600

Capture d’écran 2022-02-10 à 13 09 00

**Comment by tiroj** [comment]
Yes, looks like missing U+25CC glyph issue.

File: ofl/anektelugu/METADATA.pb [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+  tag: "wdth"
+  min_value: 75.0
+  max_value: 125.0
+}
+axes {
+  tag: "wght"
+  min_value: 100.0
+  max_value: 800.0
+}
+languages: "aa_Latn"
```
Comment (m4rc1e) [inline ofl/anektelugu/METADATA.pb:29]: Which version of gftools are you using?

Each of these language fields should have a comment.

Comment (yanone) [reply ofl/anektelugu/METADATA.pb:29]: I will check. Please halt all Anek reviewing anyways. I'll submit PR amendments later today for different reasons

Comment (yanone) [reply ofl/anektelugu/METADATA.pb:29]: I checked: I was using the latest github version of gftools. I confirmed with the latest pypi version, too. Both have no language comment

**Comment by yanone** [comment]
Updated

## Anek Telugu: Version 1.003 added

---
 d05b7d7: [gftools-packager] Anek Telugu: Version 1.003 added

* Anek Telugu Version 1.003 taken from the upstream repo https://github.com/EkType/Anek at commit https://github.com/EkType/Anek/commit/34074c6b406f4112e20c7ad10254a6e954d0324b.

 60b4055: [gftools-packager] ofl/anektelugu remove METADATA "source".  google/fonts#2587

**Comment by yanone** [comment]
Updated

## Anek Telugu: Version 1.003 added

---
 69ecb5b: [gftools-packager] Anek Telugu: Version 1.003 added

* Anek Telugu Version 1.003 taken from the upstream repo https://github.com/EkType/Anek at commit https://github.com/EkType/Anek/commit/34074c6b406f4112e20c7ad10254a6e954d0324b.

**Pull request merged**